tensorboard/webapp/metrics/views/card_renderer/run_name_component.ts

```diff
@@ -14,12 +14,20 @@ limitations under the License.
 ==============================================================================*/
 import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
 
+import {ExperimentAlias} from '../../../experiments/types';
+
 @Component({
   selector: 'card-run-name-component',
-  template: '{{ name }}',
+  template: `<tb-experiment-alias
+      *ngIf="experimentAlias != null"
+      [alias]="experimentAlias!"
+    ></tb-experiment-alias>
+    <span *ngIf="experimentAlias != null">/</span>
```

Comment:
Normally, in an inline element, every whitespace counts so Please try to remove whitespace and newlines. 
```
</tb-experiment-alias><span
  *ngIf="experimentAlias != null">/</span>
```

Generally, Prettier is pretty good at doing the right thing.